src/main/java/uk/gov/companieshouse/api/accounts/filter/AccountsNoteFilter.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.companieshouse.api.accounts.filter;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.WordUtils;
```

Comment:
Unused imports . Would help to run sonar lint locally so that these can be caught.

Reply:
Sorry! Good catch 👍 removed